test/tests/haskell-ghci/container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+echo "" | ghci
```

Comment:
@Moghedrin this line is confusing me.  What's this actually doing, especially with that "unnecessary use of double quotes"?